### PR TITLE
feat/DOP 47/ui tabs

### PR DESCRIPTION
**Pull request by garyunwin42**
This PR introduces `Tabs` in the Storybook. Headless UI does not support scrollable `Tabs` and hence was not implemented.

### CODE REVIEW

**Comment by witherblock** [comment]
@garyunwin42 any updates on this?

**Comment by garyunwin42** [comment]
> @garyunwin42 any updates on this?

fixed. thanks!

**Review by witherblock — CHANGES_REQUESTED** [review]
No export in index file

File: packages/ui/src/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        {children}
+      </HeadlessTab.List>
+      <HeadlessTab.Panels className={`my-2 ${SIZES[size]}`}>
+        <HeadlessTab.Panel className="rounded-lg bg-umbra text-white p-1">
+          {size}
+        </HeadlessTab.Panel>
+      </HeadlessTab.Panels>
+    </HeadlessTab.Group>
+  );
+}
```
Comment (witherblock) [inline packages/ui/src/Tabs.tsx:65]: Define this component like the above one, separate type and big arrow function and props broken down in the function

**Review by witherblock — CHANGES_REQUESTED** [review]

**Review by witherblock — APPROVED** [review]

**Pull request merged**